### PR TITLE
Mark two more tests as new opmath only

### PR DESCRIPTION
**Pull request by astralcai**

### CODE REVIEW

**Review by dwierichs — APPROVED** [review]

File: tests/devices/default_tensor/test_scalability.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ def circuit():
 
         _ = qml.QNode(circuit, dev)()
 
+    @pytest.mark.usefixtures("new_opmath_only")
```
Comment (Qottmann) [inline tests/devices/default_tensor/test_scalability.py:129]: Do you know why this fails with old opmath?

Comment (albi3ro) [reply tests/devices/default_tensor/test_scalability.py:129]: potentially the prod?

**Review by PietropaoloFrisoni — APPROVED** [review]

File: tests/devices/default_tensor/test_scalability.py (hunk 1 above)
Comment (Qottmann) [reply tests/devices/default_tensor/test_scalability.py:129]: Oh yes! Perhaps the better way to resolve this is to define the Hamiltonian as
```
qml.Hamiltonian([1.], [reduce(lambda a, b: a @ b, (Y(i) for i in range(wires)))])
```
?

Comment (astralcai) [reply tests/devices/default_tensor/test_scalability.py:129]: There is a separate test case with `Tensor` right above this one.

Comment (Qottmann) [reply tests/devices/default_tensor/test_scalability.py:129]: Oh all good then 👍

**Review by Qottmann — APPROVED** [review]

**Pull request merged**